src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -3,10 +3,12 @@ import { FormControl, Validators } from '@angular/forms';
 import { ActivatedRoute, Router } from '@angular/router';
 
 import * as _ from 'lodash';
-import { forkJoin } from 'rxjs';
+import { BsModalService } from 'ngx-bootstrap/modal';
+import { forkJoin, Subscription } from 'rxjs';
 
 import { ErasureCodeProfileService } from '../../../shared/api/erasure-code-profile.service';
 import { PoolService } from '../../../shared/api/pool.service';
+import { DeletionModalComponent } from '../../../shared/components/deletion-modal/deletion-modal.component';
```

Comment:
Got the following error message when running ``npm start``:
```
Date: 2018-10-29T13:27:18.008Z
Hash: 6d56854c0816f59aff3f
Time: 9670ms
chunk {main} main.js, main.js.map (main) 1.93 kB [initial] [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 685 bytes [initial] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 5.22 kB [entry] [rendered]
chunk {scripts} scripts.js, scripts.js.map (scripts) 536 kB  [rendered]
chunk {styles} styles.js, styles.js.map (styles) 292 kB [initial] [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 325 kB [initial] [rendered]

ERROR in src/app/ceph/pool/pool-form/pool-form.component.ts(11,40): error TS2307: Cannot find module '../../../shared/components/deletion-modal/deletion-modal.component'.

ℹ ｢wdm｣: Failed to compile.
```

Reply:
Component has been renamed, see https://github.com/ceph/ceph/commit/e8218ac14590cdcc36d1e93120b9c77a58108e47#diff-89efc18d06edcfa1553c63af45784e9c